views/components/Matrix.d3.js

```diff
@@ -61,7 +61,7 @@ export default class Matrix {
       .select(this.el)
       .append('svg')
       .attr('width', () => Math.max(this.width, 0))
```

Comment:
I think this line does need to change to `.attr('width', '100%')` right?

Reply:
That line has been kept intact when we've been testing for the solution, so this can stay imo.